### ISSUE TITLE
How can I get my own expert tajectory from a pretrained model?

### ISSUE THREAD

**Issue by wang88256187** [ikostrikov/pytorch-a2c-ppo-acktr-gail#217]

**Comment by wang88256187** [comment]
I want to generate trajectory by my own model, how can I get the original obs (not the running state which may help train)? Thank you!

**Comment by lviano** [comment]
Hi! Did you find a solution for the problem? I would like to do the same

Cheers
Luca

**Comment by AAgha66** [comment]
You can use env.get_original_obs() which gets the original version of the latest observation and not the normalized one
